easybuild/easyconfigs/n/NAMD/NAMD-2.11-CrayIntel-2016.11-pat-6.4.3.eb

```diff
@@ -0,0 +1,36 @@
+# Contributed by gppezzi, Luca Marsella and Christopher Bignamini (CSCS)
+easyblock = 'MakeCp'
+
+name = 'NAMD'
+version = '2.11'
+patversion = "643"
+versionsuffix = '-pat-%s' %(patversion)
+
+homepage = 'http://www.ks.uiuc.edu/Research/namd/'
+description = """NAMD is a parallel molecular dynamics code designed for high-performance simulation of large biomolecular systems."""
+
+toolchain = {'name': 'CrayIntel', 'version': '2016.11'}
+
+sources = ['NAMD_%(version)s_Source.tar.gz']
+
+
+prebuildopts = ' ./config CRAY-XC-intel Eos --with-fftw3 --tcl-prefix $EBROOTTCL --charm-base $EBROOTCHARMPLUSPLUS --charm-arch gni-crayxc-smp --cxx-opts  "$CRAY_UGNI_POST_LINK_OPTS -lugni $CRAY_PMI_POST_LINK_OPTS -lpmi" && '
+prebuildopts += ' cd CRAY-XC-intel && '
+
+builddependencies = [
+    ('fftw/3.3.4.10', EXTERNAL_MODULE),
+    ('rca', EXTERNAL_MODULE),
+    ('craype-hugepages8M', EXTERNAL_MODULE),
+    ('Charm++', '6.7.1'), 
+    ('Tcl', '8.5.9', '-namd',  True),
+    ('perftools-cscs/%snogpu' %patversion, EXTERNAL_MODULE),
```

Comment:
@ChristopherBignamini perftools is now provided by easybuild, so instead of using EXTERNAL_MODULE you can now use:
```
 ('perftools-cscs', '%snogpu' %patversion),
```